src/brag/client/ceph-brag

```diff
@@ -258,7 +258,7 @@ def get_nums():
   num_mons = len(oj['monmap']['mons'])
   num_osds = int(oj['osdmap']['osdmap']['num_in_osds'])
   try:
-    num_mdss = oj['mdsmap']['in']
+    num_mdss = oj['fsmap']['in']
```

Comment:
this is introduced by 4e9b953. @jcsp is it okay to use fsmap here? if yes, shall we backport this to jewel and kraken?